ops/framework.py

```diff
@@ -1093,6 +1110,8 @@ def __eq__(self, other):
             return self._under == other._under
         elif isinstance(other, collections.abc.Sequence):
             return self._under == other
+        elif isinstance(other, list):
+            return list(self._under) == other
```

Comment:
This seems redundant; isinstance([], collections.abc.Sequence) returns true, thus the existing clause should catch actual lists as well.  I'd drop the separate if for explicit lists here, as well as the other added occurrences.